src/python/pants/engine/interactive_runner.py

```diff
@@ -21,6 +22,7 @@ class InteractiveProcessResult:
 class InteractiveProcessRequest:
   argv: Tuple[str, ...]
   env: Tuple[str, ...] = ()
+  input_files: Digest = EMPTY_DIRECTORY_DIGEST
```

Comment:
Should this be `Optional[Digest]`? I think the answer is no because of how Rust is implemented.

Reply:
Yeah we don't have an abstraction right now to easily handle passing Python types with "optional" semantics (i.e. a value that we expect to maybe be `None` or maybe be some specific Python type) over to Rust such that they wind up as a Rust `Option<T>`. We could possibly build this, but we haven't yet, and it's easier to just assume this value is always a `Digest` and use the `EMPTY_DIRECTORY_DIGEST` sentinal value we already have.